libs/LedKit/include/animations/BlinkOnCharge.h

```diff
@@ -45,6 +45,13 @@ class BlinkOnCharge : public interface::LEDAnimation
 	uint8_t _step		  = 0;
 	uint8_t _stage		  = 0;
 
+	void stage0();
+	void stage1();
+	void stage2();
```

Comment:
can you name them something meaningful as you only have 3 and they do specific things